### ISSUE TITLE
"js -r script-doc.js" Fails if filename contains "-d"

### ISSUE THREAD

**Issue by mozbugbox** [tridactyl/tridactyl#3684]
When load a `js` with a js file which has a "-d" in its filename, a error emitted:

```
# controller in excmd: TypeError: str.join(...).split(...)[1] is undefined
```

It seems the `-d` interfere somehow with args parsing for the `js` command.

JS File: [save-document.txt](https://github.com/tridactyl/tridactyl/files/6487484/save-document.txt)

**Comment by bovine3dom** [comment]
https://github.com/tridactyl/tridactyl/blob/c3f86b6df252e3aab28295e4a2a3fae26b41ce28/src/excmds.ts#L4940

Yeah this is a daft bug, it should check `if flag starts with "-d"` like all the others do

**Comment by mozbugbox** [comment]
so changed to
```ts
        const match = /^-d(.)/.exec(flag)
```

**Comment by bovine3dom** [comment]
Yes exactly I think. Feel free to submit a PR if I don't get round to it soon :)
